packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -132,19 +159,36 @@ export class ComponentUtils {
     );
 
     let componentsList: string[];
+    const connection = await Connection.create({
+      authInfo: await AuthInfo.create({
+        username: await OrgAuthInfo.getUsername(defaultOrg)
```

Comment:
Because of the issue we saw yesterday, perhaps a bounds check should be added at line 155, and a guard against defaultOrg being undefined or empty should be added. (and throw an exception if it's missing)